src/dbg/module.cpp

```diff
@@ -105,6 +105,20 @@ bool ModLoad(duint Base, duint Size, const char* FullPath)
     info.fileMap = nullptr;
     info.fileMapVA = 0;
 
+    // Determine whether the module is located in system
+    wchar_t sysdir[MAX_PATH];
+    GetEnvironmentVariableW(L"windir", sysdir, sizeof(sysdir));
+    String Utf8Sysdir = StringUtils::Utf16ToUtf8(sysdir);

```

Comment:
Maybe we should add to lowercase here, just to be sure? Maybe there are some rare cases where windows reports a different case?